src/lib/server.ts

```diff
@@ -233,6 +238,36 @@ export class ClientsController {
       this.eventForwarder = new EventForwarder(this);
       this.eventForwarder.start();
     }
+
+    if (this.loggingEventForwarder === undefined) {
+      this.loggingEventForwarder = new LoggingEventForwarder(this, this.driver);
+    }
```

Comment:
No need to initialize it until we start it.

Reply:
(I know you're following the event forwarding stuff, but that is probably also up for a refactor now that listening is also opt-in and people might just check the supported versions)